build/docker/entrypoint_local.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
Can't you use the normal entrypoint script - how is this one different?

Reply:
it sets remote debugging when starting the JVM